prototype/galore/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
```

Comment:
I think we can rework the exisitng setup.py to package your kernels into the core package - Happy to credit you in the files directly and/or README

Reply:
np -- whatever is easiest